examples/mobile/HomeApp/js/app.js

```diff
@@ -74,6 +74,38 @@ import Storage from "./clipping-storage.js";
 				})
 		});
 
+	async function validateAppsList() {
+		let app,
+			webClip;
+
+		for (app of appsList) {
+			for (webClip of app.webClipsList) {
+				try {
+					// we wait for the every fetch response so validateAppsList
+					// will return when all job is done, not earlier
+					const response = await fetch(webClip.url + "\\webclip.html");
```

Comment:
I think you should try to do this simultaneously

Reply:
I used Promise.allSettled i new approach